docs/reference/contributing/guidelines/workflow.md

```diff
@@ -61,90 +61,131 @@ If commits do not follow the above guidelines, we may request that you modify th
 
 [How We Release Arm Mbed OS](../introduction/how-we-release-arm-mbed-os.html) explains Mbed OS versioning.
 
-## Pull request types
+## Pull request template
```

Comment:
The blog post says the template now "includes a note about reporting and questions and enhancements to the Forum," but I don't see that here.

Reply:
Ah I think the blog post might have been a little confusing. When someone raises an issue that is not a bug then the git2jira bot closes the issue with a comment stating that questions or enhancements should be raised in the forums. It has nothing to do with PRs

Reply:
In the issue template:
(which is not affected by this PR)
************************************** WARNING **************************************

   The ciarcom bot parses this header automatically. Any deviation from the 
   template may cause the bot to automatically correct this header or may result in a 
   warning message, requesting updates.

   Please ensure all sections of the template below are filled in and no changes 
   are made to the template format. Only bugs should be raised here as issues. 
   Questions or enhancements should instead be raised on our forums:
   https://forums.mbed.com/ .

   *************************************************************************************

Reply:
Thanks for clarifying 👍 
Should we mention somewhere (not in the PR section) that nonbug issues will be closed?

Reply:
> You can submit Mbed OS bugs on the forums or directly on GitHub.

I'll send separate PR to change this in this workflow file

Reply:
Here https://github.com/ARMmbed/mbed-os-5-docs/pull/1161